packages/typescript/rules-snapshot.json

```diff
@@ -66,6 +66,7 @@
   "constructor-super": "off",
   "default-param-last": "off",
   "getter-return": "off",
+  "import/named": "off",
```

Comment:
:eyes: This doesn't seem right

Reply:
Ah https://github.com/benmosher/eslint-plugin-import/blob/master/config/typescript.js#L25

Reply:
Au contraire! https://github.com/benmosher/eslint-plugin-import/blob/6c8981d/config/typescript.js#L25-L27